<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {    
	background-color: qlineargradient(spread:pad, x1:1, y1:0.539, x2:0, y2:0, stop:0 rgba(53, 53, 53, 255), stop:1 rgba(98, 98, 98, 255));
    color: rgba(255, 255, 255, 192);
	border: none;
}

QLineEdit {
    background-color: rgba(255, 255, 255, 32);
	border: 2px solid rgba(98, 98, 98, 64);
	border-radius: 2px;
}

QLineEdit::hover {
    background-color: rgba(255, 255, 255, 37);
}

QLabel {
	background-color: transparent;
    font: bold;
}

QPushButton {
	background-color: qlineargradient(spread:pad, x1:1, y1:0.539, x2:0, y2:0, stop:0 rgba(78, 78, 78, 255), stop:1 rgba(108, 108, 108, 255));
	border: 2px solid rgba(98, 98, 98, 64);
	border-radius: 2px;
    padding-left: 8px;
    padding-right: 8px;
}

QPushButton::hover {
	background-color: qlineargradient(spread:pad, x1:1, y1:0.539, x2:0, y2:0, stop:0 rgba(81, 81, 81, 255), stop:1 rgba(118, 118, 118, 255));
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="spacing">
     <number>8</number>
    </property>
    <item row="5" column="1">
     <widget class="QLabel" name="enterGuessLabel">
      <property name="text">
       <string>Enter Guess</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <spacer name="bottomSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="0">
     <spacer name="leftSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="2">
     <spacer name="rightSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="titleLabel">
      <property name="styleSheet">
       <string notr="true">font-size: 16px;</string>
      </property>
      <property name="text">
       <string>MASTERMIND</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <layout class="QHBoxLayout" name="buttonHBox">
      <item>
       <widget class="QPushButton" name="newGameBtn">
        <property name="text">
         <string>New Game</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="guessBtn">
        <property name="text">
         <string>Guess</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <spacer name="topSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="feedback">
      <property name="styleSheet">
       <string notr="true">font-size: 16px;
font: basic;</string>
      </property>
      <property name="text">
       <string>****</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <spacer name="centerSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>585</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
